prism/src/main/java/org/apache/falcon/jdbc/MonitoringJdbcStateStore.java

```diff
@@ -67,82 +70,88 @@ public MonitoredFeedsBean getMonitoredFeed(String feedName){
         } finally {
             entityManager.close();
         }
-        return ((MonitoredFeedsBean)result.get(0));
+        return ((MonitoredEntityBean)result.get(0));
     }
 
-    public void deleteMonitoringFeed(String feedName) {
+    public void deleteMonitoringFeed(String entityName, String entityType) {
```

Comment:
In this and all other methods, it is better to use EntityType enum in the method signatures.